src/modal/peraWalletConnectModalUtils.tsx

```diff
@@ -74,6 +79,59 @@ function removeModalWrapperFromDOM(modalId: string) {
   }
 }
 
+function getPeraConnectModalAccordionData(uri: string): AccordionData[] {
```

Comment:
Do we need a function for this? We might want to use it as a const

Reply:
We need uri for generate QRCode. I create function for this.